data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/ExpressionEvaluationException.java

```diff
@@ -0,0 +1,11 @@
+package org.opensearch.dataprepper.expression;
```

Comment:
I think this file is missing the copyright text

Reply:
It is still missing.